From b6c2c157f403c9ab74067cc2611ffebd3f87d8d7 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Fri, 5 Aug 2022 10:37:09 +0200
Subject: [PATCH] Drop bogus dependencies

This reverts 90d78bd238c8db701e4bb19d9c5c2ddd9bd4f59c,
93374114e87a9b05546fb27bc936f223b5b84f7d and
92ef8b8ef8d0c5229fc601ac36806c926c85c36a.

The proper fix went into alkimia via
https://invent.kde.org/office/alkimia/-/commit/09a11cee26af4ca55e5839dd44e32400147dc119

See also: https://invent.kde.org/office/kmymoney/-/issues/63

Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 842acedaa..5a4542d29 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -110,7 +110,7 @@ if (PkgConfig_FOUND)
 endif()
 
 find_package(Qt${QT_MAJOR_VERSION} ${QT_MIN_VERSION} REQUIRED COMPONENTS Core DBus Widgets Svg Xml Test PrintSupport)
-find_package(Qt${QT_MAJOR_VERSION} ${QT_MIN_VERSION} OPTIONAL_COMPONENTS Sql Concurrent QuickWidgets WebEngineWidgets WebKitWidgets)
+find_package(Qt${QT_MAJOR_VERSION} ${QT_MIN_VERSION} OPTIONAL_COMPONENTS Sql Concurrent QuickWidgets)
 
 find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS Archive CoreAddons Config ConfigWidgets I18n Completion KCMUtils ItemModels ItemViews Service XmlGui TextWidgets Notifications KIO)
 
-- 
2.40.0

